ops/terraform/services/base/main.tf

```diff
@@ -1,18 +1,89 @@
+module "terraservice" {
+  source = "../_modules/bfd-terraservice"
+
+  environment_name     = terraform.workspace
+  relative_module_root = "ops/terraform/services/base"
+}
+
 locals {
+  account_id       = data.aws_caller_identity.current.account_id
   default_tags     = module.terraservice.default_tags
   env              = module.terraservice.env
   seed_env         = module.terraservice.seed_env
   is_ephemeral_env = module.terraservice.is_ephemeral_env
-  kms_key_alias    = "alias/bfd-${local.seed_env}-cmk"
-}
 
-module "terraservice" {
-  source = "../_modules/bfd-terraservice"
-
-  environment_name     = terraform.workspace
-  relative_module_root = "ops/terraform/services/base"
+  kms_key_alias = "alias/bfd-${local.seed_env}-config-cmk"
+  key_policy_template = jsonencode(
+    {
+      Version = "2012-10-17",
+      Id      = "config-key-policy",
+      Statement = [
+        # This policy statement is the default key policy allowing the account principal to delegate
+        # permissions, via IAM policies, to other principals within the AWS Account. Due to this
+        # delegation, it is not necessary to include additional IAM User/Group-specific policy
+        # statements within this key policy, as that permission delegation can be handled via
+        # distinct IAM Policies attached to those entities. See
+        # https://docs.aws.amazon.com/kms/latest/developerguide/key-policy-default.html#key-policy-default-allow-root-enable-iam
```

Comment:
Thank you. The key policy/IAM policy divide is a bit of a mess. I'm heartened to see that we're not extending the mess.